app/assets.js

```diff
@@ -1,101 +1,102 @@
-var config = require('./config');
-var log = require('bole')('assets');
-var path = require('path');
+const Bluebird = require('bluebird');
+const fs = Bluebird.promisifyAll(require('fs'));
+const glob = Bluebird.promisify(require('glob').glob);
+const render = Bluebird.promisify(require('node-sass').render);
+const concatfiles = require('concat-files');
+const path = require('path');
 
-var Promise = require('bluebird');
-var fs = Promise.promisifyAll(require('fs'));
-var glob = Promise.promisify(require('glob').glob);
-var render = Promise.promisify(require('node-sass').render);
-var concatfiles = require('concat-files');
+const config = require('./config');
+const log = require('bole')('assets');
 
 
-exports.compile_sass = function () {
+exports.compile_sass = () => {
 
-  config.sass.sources.forEach(function (options) {
+  config.sass.sources.forEach((options) => {
```

Comment:
You already know this because you did it in a different part of this PR, but you could omit the parenthesis around options as the arrow function has only one argument.

I still have to make up my mind about this "special" case as parenthesis are mandatories if function has 0 or 2+ arguments - but I admit that it's nice to not have the double parenthesis in cases like this or when function is very small so I'd probably remove them if optional.

Reply:
The more I think about it and look at this PR the more I think omit the parenthesis when the arrow function only take only one argument makes it more readable (and we're already doing this).

Reply:
I'm also undecided on the parenthesis. Double parentheses are ugly, but removing them contributes to the same issue as omitting braces on a single line `if` block. I think we should always use the parentheses even with only 1 argument.